### ISSUE TITLE
How should we handle studio thumbnails with no user-set thumbnail?

### ISSUE THREAD

**Issue by jwzimmer** [LLK/scratch-www#1228]
**TL;DR:** We pull the first project’s thumbnail if no studio thumbnail is provided. @mewtaylor said "there’s no thumbnail associated with the studio data. [...] if we don’t get a studio thumbnail, we fall back to the first project, but don’t add that thumbnail to the studio in our data." As a result, if your search results contain studios that never had a manually-set thumbnail, currently you will see the default thumbnail, rather than the project thumbnail you will see on that studio's page.

One solution could be adding that project thumbnail as the studio thumbnail, and treating it just like a user-set thumbnail.

Relevant to https://github.com/LLK/scratchr2/pull/4434, but I put it in scratch-www because the most obvious place this issue is visible is the Search page > studio results.

**Comment by rschamp** [comment]
I like the idea of just uploading the project thumbnail as the studio thumbnail. One tricky thing will be knowing whether to actually upload the project thumbnail or not, since ideally we shouldn't check S3 to find out.  We could start using the thumbnail field for this purpose though:
 * On Studio creation, set the thumbnail field to blank
 * On manual thumbnail upload, set the thumbnail field to the thumbnail filename
 * On first project add, check if the studio thumbnail field is blank, and if so upload the project's thumbnail to the studio thumbnail's location (but leave the studio thumbnail field as blank).

**Comment by rschamp** [comment]
Moved to LLK/scratch-search#31
